### DISCUSSION TITLE
v2 的查询列表有具体示例吗？

### DISCUSSION THREAD

**Discussion by DremyGit** [alibaba/formily#1807] [Q&A 2.x] [answered]
官网的案例页面 https://v2.formilyjs.org/zh-CN/guide/scenes/query-list 提到 AList，但 AList 只支持 formily v1，不支持 v2，所以目前如果想用 formily v2 做查询列表有什么现成的示例吗？

**Comment by janryWang** [comment, marked as answer]
alist后续不维护了，查询列表场景你就自己用formily查到数据渲染table就行了
